iphone/Classes/CalendarModule.m

```diff
@@ -244,7 +244,9 @@ -(void) requestAuthorization:(id)args forEntityType:(EKEntityType)entityType
                                  completion:^(BOOL granted, NSError *error){
                                      NSDictionary* propertiesDict;
                                      if (error == nil) {
-                                         propertiesDict = [TiUtils dictionaryWithCode:(granted ? 0 : 1) message:nil];
+                                         NSString* errorMsg = (granted ? nil : @"The user has denied access to events in Calendar.");
+                                         propertiesDict = [TiUtils dictionaryWithCode:(granted ? 0 : 1)
```

Comment:
Check the ``propertiesDict``, the TiUtils call is shifted (http://abload.de/img/bildschirmfoto2015-12p0jkn.png)